### PR TITLE
XDR-IOCS - Push IOCs PB edge case fix

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-34855)

## Description
Fixes an edge case when the quers finds only hashes that are not supported by XDR.

The playbook will now check if it has any IOCs that needs to be pushed after the triage.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/6_1_33.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR IOCs - Push new IOCs to XDR
+
+- Fixed an issue with the playbook failing when only hashes are found but non are valid for XDR.
```
Comment (ssokolovich) [inline Packs/CortexXDR/ReleaseNotes/6_1_33.md:6]:
```suggestion
- Fixed an issue with the playbook failing when only hashes are found but none are valid for XDR.
```

**Comment by melamedbn** [comment]
@Ni-Knight Please update the playbook image as well.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml [hunk 2]
```diff
@@ -945,14 +945,58 @@ tasks:
...
+      "yes":
+      - "7"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: ${iocs_to_push}
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml:972]: add unique here as well pls 🙏

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml [hunk 3]
```diff
@@ -934,8 +900,154 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 4e43a563-6013-49ac-8e0e-c9e03dcc1e2e
+    type: condition
+    task:
+      id: 4e43a563-6013-49ac-8e0e-c9e03dcc1e2e
+      version: -1
+      name: Indicators to push?
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml:921]:
```suggestion
      name: Are there indicators to push?
```

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml [hunk 4]
```diff
@@ -934,8 +900,154 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "46":
+    id: "46"
+    taskid: 456ed20a-e2a2-4fed-8b6d-87168c6b62a6
+    type: condition
+    task:
+      id: 456ed20a-e2a2-4fed-8b6d-87168c6b62a6
+      version: -1
+      name: Save irrelevant hashes in context for marking
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml:998]:
```suggestion
      name: Are there any irrelevant hashes in the context for marking?
```

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml [hunk 5]
```diff
@@ -934,8 +900,154 @@ tasks:
...
+    nexttasks:
+      '#default#':
+      - "45"
+      "yes":
+      - "41"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml:1012]: change this pls to not empty

**Review by ssokolovich — COMMENTED** [review]
some notes :)

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**